app/controllers/api/groups_controller.rb

```diff
@@ -1,13 +1,18 @@
 class API::GroupsController < API::RestfulController
   load_and_authorize_resource only: :show, find_by: :key
   load_resource only: [:upload_photo, :use_gift_subscription], find_by: :key
-  skip_before_action :authenticate_user!, only: :index
+  skip_before_action :authenticate_user!, only: [:index, :count_explore_results]
```

Comment:
Hey @HSalmon did you find you needed this? I think we actually want to remove this line, because we're not actually calling `authenticate_user` as a before filter in this controller anywhere afaict.